src/Lime.Transport.WebSocket/WebSocketTransport.cs

```diff
@@ -254,16 +252,15 @@ protected override async Task PerformCloseAsync(CancellationToken cancellationTo
             }
         }
 
-        public override bool IsConnected => WebSocket.State
-            >= WebSocketState.Open && WebSocket.State <= WebSocketState.CloseReceived; // We need to consider the Close status here to make the channel call the CloseAsync method.
+        public override bool IsConnected => WebSocket.State == WebSocketState.Open ||
+                                            WebSocket.State == WebSocketState.CloseReceived; // We need to consider the CloseReceived status here to make the channel call the CloseAsync method.
 
         public override string LocalEndPoint
         {
             get
             {
                 try
                 {
-#if NETSTANDARD2_0
                     try
                     {
```

Comment:
More pre-netstandard2.1 checks that can be removed?

Reply:
No, because the WS can still be running in net framework 4.8 which supports netstandard2.1